Packs/PingIdentity/Integrations/PingOne/PingOne_test.py

```diff
@@ -0,0 +1,184 @@
+from PingOne import Client, get_user_command, create_user_command, get_groups_for_user_command
+import pytest
```

Comment:
Why do you write UTs only for these commands? what about the others?

Reply:
The new TestPingOnePlaybook runs all the commands.